generator/views/glean_ping_view.py

```diff
@@ -29,6 +29,16 @@
     "rate",
     "timespan",
     "uuid",
+    "url",
```

Comment:
looks like we need `labeled_rate` and `text` here

Reply:
`labeled_rate` does not really exist (which means I should actually remove it from the other list, too)
`text` exist, but not currently used (afaik) and in general category 3 or 4. Not sure if we want to default-expose that.